spec/order_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'order'
+
+describe Order do
+
+it {is_expected.to respond_to(:sum).with(1).argument}
+
+  describe '#sum' do
+    it 'returns a sum of cost of items recieved' do
+      expect(subject.sum([{:Burger => 5.00}, {:Burger => 5.00}, {:Fish => 6.00}])).to eq 16
+    end
```

Comment:
It would be nice to have several tests for this. Otherwise, with just this one test, your method sum could return a hardcoded 16 and still work. Test with an empty food list, and at list two different non-empty food lists.